twint/get.py

```diff
@@ -108,12 +108,19 @@ def get_connector(config):
 
 async def RequestUrl(config, init):
     logme.debug(__name__ + ':RequestUrl')
+    csrf_token = random.randbytes(16).hex() # Looks like any random string works
```

Comment:
Just a note, randbytes is Python 3.9+ ([late 2020](https://www.python.org/downloads/release/python-390/). No idea if you want to support older versions, in that case it can be made to work, I guess hardcoding a fixed string might even be feasible.

Reply:
Personally, I'd like to run the code on Python 3.6. Seems as if there would be [enough options](https://stackoverflow.com/questions/2257441/random-string-generation-with-upper-case-letters-and-digits) available though ...

Reply:
This is definitely feasible.
It's just that Python 3.6 is already 7 years old, and already EOL. 3.7 is going to be EOL'd in 6 weeks according to https://devguide.python.org/versions/. Debian Bullseye (current stable) ships 3.9, and I usually find that to be a reasonable reference to set the cutoff point.

Reply:
@LinqLover Python 3.6 has already reached end of support, and Python 3.7 reaches end of support on 2023-06-27 (1 month 17 days away). TWINT should not care about Python versions that have reached end of support.

Reply:
Fair point :+1: